pyproject.toml

```diff
@@ -22,18 +22,18 @@ classifiers = [
     "Programming Language :: Python :: 3.11",
     "Programming Language :: Python :: 3.12",
     "Framework :: Django",
-    "Framework :: Django :: 3.2",
     "Framework :: Django :: 4.2",
     "Framework :: Django :: 5.0",
     "Framework :: Wagtail",
     "Framework :: Wagtail :: 4",
-    "Framework :: Wagtail :: 5"
+    "Framework :: Wagtail :: 5",
+    "Framework :: Wagtail :: 6"
 ]
 dynamic = ["version"]
 requires-python = ">=3.8"
 dependencies = [
-    "Django>=3.2,<5.1",
-    "Wagtail>=5.2",
+    "Django>=4.2,<5.1",
+    "Wagtail>=5.2,<6.3",
```

Comment:
Please leave the upper boundary off.
Also, Wagtail 6.2 supports Django 5.1

Reply:
Thanks, updated in a25d279